### ISSUE TITLE
Removed Plex Library while rules were attached to that library

### ISSUE THREAD

**Issue by ydkmlt84** [Maintainerr/Maintainerr#1302]
I removed a library from inside of Plex that was attached to multiple rules. When I added the library back, with a new Library section ID (but the same media), Maintainerr was not working properly during Rule Execution. The only indication in the log was below:
```
[maintainerr] | 05/10/2024 11:14:20  [WARN] [PlexApiService] Plex api communication failure.. Is the application running?
[maintainerr] | 05/10/2024 11:14:20  [DEBUG] [PlexApiService] Cannot read properties of undefined (reading 'MediaContainer')
```
It was not immediately clear why my rules were giving me that error, until I looked in the Plex logs and found this:
```
Oct 05, 2024 15:09:57.668  DEBUG - Request: GET /library/sections/16/all?includeGuids=1&type=1 (12 live) #159a TLS Page 0-49 GZIP Signed-in Token (ydkmlt84) (129.0.0.0)
Oct 05, 2024 15:09:57.668 WARN - [Req#159a] Invalid library section ID 16 passed.
```
Which led me to think about my "new" library and then realize that I needed to update the rules' library. Then I went to my rules and found that each Rule that had that old library attached, was now showing "undefined".

A better log indication would be more helpful since there wasn't an actual communication issue with the Plex API, which is where I initially looked.

**Comment by jorenn92** [comment]
Good point, Maintainerr won't notice the ID change. I agree a better log message would help a lot in this case.

**Comment by enoch85** [comment]
@ydkmlt84 Is this still valid?

**Comment by ydkmlt84** [comment]
I am not sure about the logs and I would have to recreate the scenario. The thing that happened though, I'm sure is still a thing and I don't see a way around it.

I am not sure when I'll be able to try and recreate but I'll try.
